aospy/model.py

```diff
@@ -74,9 +74,9 @@ def _get_grid_files(self):
         datasets = []
         for path in self.grid_file_paths:
             try:
-                ds = xray.open_dataset(path)
```

Comment:
Without the ```decode_cf=False``` argument this line throws an exception in idealized cases whose time units are specified as "days since 0000-00-00 00:00:00", because no library in python supports dates with year 0.

I think we might be able to relax this somewhat by setting ```decode_time``` to ```False``` instead.  I'll give it a try.  Do you see any harm in adding the ```decode_time``` or ```decode_cf``` arguments here?  My impression is that it doesn't impact the calculations, but safeguards against some mysterious errors.